Code/Mantid/Framework/PythonInterface/plugins/algorithms/RefLReduction.py

```diff
@@ -375,8 +397,10 @@ def PyExec(self):
                                                            final_y_error_axis,
                                                            name_output_ws,
                                                            ws_event_data)
+        AddSampleLog(Workspace=name_output_ws,
+                     LogName='isSFfound',
+                     LOgText=str(isSFfound))
```

Comment:
Should the log name be something different (e.g. `isSFFound`) to follow our naming conventions?